impeller/tessellator/tessellator.cc

```diff
@@ -53,6 +54,8 @@ VertexBuffer Tessellator::TessellateConvex(const Path& path,
               index_buffer.GetBuffer()->OnGetContents() +
               index_buffer.GetRange().offset));
       path.WritePolyline(tolerance, writer);
+      point_buffer.GetBuffer()->Flush(point_buffer.GetRange());
+      index_buffer.GetBuffer()->Flush(index_buffer.GetRange());
```

Comment:
I don't have a test case for this, except for the fact that it may or may not work on swiftshader. I don't want to add a mock test for these, instead I think switching to the point arena like we did for strokes is probably better?

Reply:
So, you're saying if this wasn't here the goldens would look wrong?  Sounds good to me.

Reply:
It would be racy :(